includes/functions.php

```diff
@@ -165,7 +165,7 @@ function swedbank_pay_get_order_lines( WC_Order $order ) {
 			Swedbank_Pay_Order_Item::FIELD_CLASS       => $product_class,
 			Swedbank_Pay_Order_Item::FIELD_ITEM_URL    => $product->get_permalink(),
 			Swedbank_Pay_Order_Item::FIELD_IMAGE_URL   => $image_url,
-			Swedbank_Pay_Order_Item::FIELD_DESCRIPTION => $product->get_description(),
+			Swedbank_Pay_Order_Item::FIELD_DESCRIPTION => $product->get_name(), // limited to 40 characters.
```

Comment:
Suggestion: If this is limited to 40 characters, should we trim it in that case? And are we sure we should be using the name here?

I'm not sure if Swedbank might want the actual description here or not, but it might be good to just check what this is used for.